app/views/RoomView/RightButtons.tsx

```diff
@@ -476,17 +521,30 @@ class RightButtonsContainer extends Component<IRightButtonsProps, IRigthButtonsS
 						disabled={hasE2EEWarning}
 					/>
 				) : null}
-				{rid ? <HeaderCallButton rid={rid} disabled={hasE2EEWarning} /> : null}
+				{rid ? (
```

Comment:
There's a `!rid` with return null right above.
It's not part of this task's scope, but remove the `rid` check here and see if there's any issue.
I think it's worth trying.